content/docs/integrations/fleet.mdx

```diff
@@ -0,0 +1,278 @@
+| `failing_policies` | string | Denies access if a host fails a specific Fleet policy. |
+| `failing_critical_policy` | boolean | Denies access if a host fails any critical Fleet policies. |
+| `failing_any_policy` | boolean | Denies access if a host fails any Fleet policy. |
+
+## Configure the FleetDM plugin
+
+This guide shows you how to:
+
+- Deploy a Fleet instance.
+- Configure the FleetDM plugin in Pomerium Enterprise.
+- Create authorization policies in Pomerium with FleetDM criteria.
+
+To complete this guide, you need:
+
+- [Pomerium Enterprise](https://www.pomerium.com/docs/enterprise)
+- [Pomerium Core](https://www.pomerium.com/docs/core)
+- A deployed [Fleet instance](https://github.com/fleetdm/fleet/tree/main/infrastructure/render)
+- A configured certificate authority (CA) and client certificate
+- An environment to install a Fleet agent
+
+:::note
+
+For the purposes of this guide, we recommend installing the Fleet agent in a VM running macOS. You can install and run a macOS VM for free with Tart. See the [Tart Quickstart](https://tart.run/quick-start/) for more information.
+
+:::
+
+### Configure FleetDM
+
+To quickly set up FleetDM, see the [Fleet deployment guide](https://github.com/fleetdm/fleet/tree/main/infrastructure/render). After you deploy a Fleet instance, you should have the following services:
+
+- Fleet web service
+- Fleet MySQL database
+- Fleet Redis service
+
+Fleet provides a URL to access the dashboard of your Fleet instance:
+
+![The FleetDM dashboard and management web console](./img/fleet/fleet-dashboard.png)
+
+### Enroll a host
+
+Fleet [gathers host information](https://fleetdm.com/guides/enroll-hosts) by installing an [osquery](https://github.com/osquery/osquery) agent (”Fleet agent”) on each host.
+
+To enroll a host, download the Fleet agent:
+
+1. In the navigation bar, select **Hosts.**
+1. Select **Add hosts**.
+1. Select the operating system of the host you want to add. (This guide assumes **macOS.**)
+1. Enter the provided command with the [Fleet command-line tool](https://fleetdm.com/guides/fleetctl).
+
+   ![Adding a Fleet host in the Fleet dashboard](./img/fleet/fleet-add-hosts.png)
+
+   Running this command generates a `fleetd` installation package:
+
+   ```bash
+   Generating your fleetd agent...
+
+   Success! You generated fleetd at /Users/<username>/fleet-osquery.pkg
+
+   To add this device to Fleet, double-click to install fleetd.
+
+   To add other devices to Fleet, distribute fleetd using Chef, Ansible, Jamf, or Puppet. Learn how: https://fleetdm.com/learn-more-about/enrolling-hosts
+   ```
+
+1. Run your macOS VM and install the `fleet-osquery.pkg`.
+
+   If you check your Fleet dashboard, you should see the VM listed as an enrolled host:
+
+   ![Reviewing the enrolled host in the Fleet dashboard](./img/fleet/fleet-enrolled-host-vm.png)
+
+1. Lastly, install the CA bundle, which includes the client certificate, on the VM where you installed the Fleet agent.
```

Comment:
Please add the command (security import ...)

Also we're not installing ÇA bundle but just the client certificate.